src/Tracker.css

```diff
@@ -1,3 +1,23 @@
 #map, .leaflet-container {
   height: 100%;
 }
+
+.leaflet-container .leaflet-control {
+  background: #00000066;
```

Comment:
This might interest you Mr. F [lab color](https://lea.verou.me/2020/04/lch-colors-in-css-what-why-and-how/)

Reply:
Yes. Thank you! The [linked article](https://lea.verou.me/2009/03/100-cyan-in-cmyk-is-not-rgb0255255/) was excellent too.